src/parser.rs

```diff
@@ -221,7 +226,7 @@ impl<'a> Parser<'a> {
                             }
                         }
                     }
-                    Some((start, ch)) if is_symbol_head(ch) => {
+                    Some((start, ch)) if ch.is_ascii_alphabetic() => {
```

Comment:
Any reason you changed this? `is_symbol_head` matches only the characters specified by the EDN spec.

Reply:
The spec specifically limits tagged elements to starting with alphabetic characters:

> edn supports extensibility through a simple mechanism. # followed immediately by a symbol **starting with an alphabetic character** indicates that that symbol is a tag.